docs/how_to/create_a_quantum_circuit.rst

```diff
@@ -0,0 +1,167 @@
+
+########################
+Create a quantum circuit
+########################
+
+This guide shows you how to create a quantum circuit.
+
+There are two ways to create a :class:`~.QuantumCircuit` object:
+
+1. Specifying the number of qubits and classical bits.
+2. Creating :class:`~.QuantumRegister`\ s and :class:`~.ClassicalRegister`\ s and use the registers to initialize a :class:`~.QuantumCircuit`
+
+Specifying the number of qubits and classical bits
+==================================================
+
+You can create a :class:`~.QuantumCircuit` by only specifying the number of qubits and classical bits. For example:
+
+.. testcode::
+
+    from qiskit import QuantumCircuit
+
+    # Initialize number of qubits and classical bits
+    n_qubits = 3
+    n_bits = 2
+
+    # Create the circuit
+    qc = QuantumCircuit(n_qubits, n_bits)
+    print(qc.draw())
+
+.. testoutput::
+    :options: +NORMALIZE_WHITESPACE
+
+    q_0: 
+        
+    q_1: 
+        
+    q_2: 
+        
+    c: 2/
+     
+
+You can also create a :class:`~.QuantumCircuit` with only qubits and no classical bits, by omitting the number of classical bits:
+
+.. testcode::
+
+    from qiskit import QuantumCircuit
+
+    qc = QuantumCircuit(n_qubits)
+    print(qc.draw())
+
+.. testoutput::
+    :options: +NORMALIZE_WHITESPACE
+
+    q_0: 
+        
+    q_1: 
+        
+    q_2: 
+
+Create from quantum and classical registers
+===========================================
+
+Create quantum registers
+------------------------
+
+You can create a :class:`~.QuantumRegister` object by passing the desired number of qubits as an argument:
+
+.. testcode::
+
+    from qiskit import QuantumRegister
+
+    # Create a quantum register with 2 qubits
+    qr1 = QuantumRegister(2)
+
+    # Create a quantum register with 3 qubits
+    qr2 = QuantumRegister(3)
+
+Create classical registers
+--------------------------
+
+Similar to the quantum registers, you can create a :class:`~.ClassicalRegister` object by passing the desired number of classical bits as an argument:
+
+.. testcode::
+
+    from qiskit import ClassicalRegister
+
+    # Create a classical register with 2 classical bits
+    cr1 = ClassicalRegister(2)
+
+    # Create a classical register with 1 classical bit
+    cr2 = ClassicalRegister(1)
+
+Initialize the quantum circuit
```

Comment:
I think it would be good to align the language used in this headings, so either:

> Creating quantum registers
...
> Creating classical registers
...
> Create a quantum circuit
...

or:

> Initializing quantum registers
...
> Initializing classical registers
...
> Initialize a quantum circuit
...